src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -4,19 +4,23 @@
 
 public class RequestOptions {
 	public static RequestOptions getDefault() {
-		return new RequestOptions(Stripe.apiKey, Stripe.apiVersion, null, null);
+		return new RequestOptions(Stripe.apiKey, Stripe.apiVersion, null, null, 30 * 1000, 80 * 1000);
```

Comment:
Would you mind declaring two new constants `DEFAULT_READ_TIMEOUT = 80*1000` and `DEFAULT_CONNECTION_TIMEOUT = 30*1000` and using the named constants here?